charts/soroban-rpc/pubnet-values.yaml

```diff
@@ -0,0 +1,224 @@
+global:
+  ## String to use to explicitly set namespace name in manifests.
+  ## Useful for those using "helm template" to render templates
+  # namespace: mynamespace
+
+  ## String to partially override common.fullname template (will maintain the release name)
+  ##
+  # nameOverride:
+
+  ## String to fully override common.fullname template
+  ##
+  # fullnameOverride:
+
+  ## List of pre-existing image Pull Secrets to use
+  # imagePullSecrets:
+  # - imagepullsecret1
+  # - imagepullsecret2
+
+  image:
+    sorobanRpc:
+      registry: docker.io
+      repository: stellar/soroban-rpc
+      tag: latest
+      ## Optional tag. 
+      ## If absent, defaults to the appVersion from Chart.yaml
```

Comment:
Maybe it's better to comment it out? Defaulting to the appVersion might be convenient. Latest can cause upgrades from underneath people so might be undesired in some environments.

Reply:
yes, https://github.com/stellar/helm-charts/pull/78/commits/fe36d488d7035ac3d9b45c28c23e85fc046e5c31